subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDoc.java

```diff
@@ -208,7 +208,6 @@ public Property<String> getMaxMemory() {
      * Optional JavaLauncher for toolchain support
      * @since 7.2
      */
-    @Incubating
     @Internal
```

Comment:
Unrelated, but I believe this should also have:
```
    @Nested
    @Optional
    ```

Reply:
I think you are right, and the launcher should be included in the task inputs, as it potentially can affect the output.

It will slightly change the behavior by making the task more sensitive to the inputs, but I think it makes sense.

Reply:
Done